Sources/xcodeproj/Objects/BuildPhase/PBXBuildPhase.swift

```diff
@@ -114,7 +114,7 @@ public extension PBXBuildPhase {
     /// - Parameter file: file element to be added to the build phase.
     /// - Returns: proxy build file.
     /// - Throws: an error if the file cannot be added.
-    public func add(file: PBXFileElement) throws -> PBXBuildFile {
+    func add(file: PBXFileElement) throws -> PBXBuildFile {
```

Comment:
Out of curiosity, why was this method made internal? I'd keep it as public.

Reply:
They are still public, they're inside a `public` extension so once you switch to the Swift 5 toolchain, it'll give a warning saying that the modifier is redundant. I'm fine with reverting these.